internal/util/projutil/project_util.go

```diff
@@ -215,8 +215,15 @@ func GetOperatorType() OperatorType {
 }
 
 func IsOperatorGo() bool {
-	_, err := os.Stat(mainFile)
-	return err == nil
+	_, errGoMod := os.Stat(goModFile)
+	_, errGopkgTOMLFile := os.Stat(gopkgTOMLFile)
+	_, errPkgApiDir:= os.Stat(pkgApiDir)
```

Comment:
Looks like you are missing a space before the `:=`

Reply:
👍